src/main/java/appeng/spatial/StorageHelper.java

```diff
@@ -139,6 +139,8 @@ private Entity teleportEntity( Entity entity, final TelDestination link )
 				passanger.startRiding( entity, true );
 			}
 		}
+		
+		entity.world.updateEntity( entity );
```

Comment:
It might be related, but it doesn't fix that issue just like that. I tried that yesterday.